release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,26 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 26 November 2024 - New feature - Admin app - Automation usage [ID 41554]
+
+From now on, the usage page in the [Admin app](https://admin.dataminer.services) will also provide usage data about Automation script runs when available.
+
+### 26 November 2024 - New feature - Usage API - Usage API with API key [ID 41554]
+
+From now on, you can create an API key on organization level with the "Retrieve usage data" permission. This API key can be used with the new Key Usage API, in combination with the new Public Usage API, to retrieve usage data about your DataMiner Systems in an automated way.
+
+The swagger documentation pages about the available Usage API calls are available in the following locations:
+
+- [Public Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Public+Usage+Api+v1.0)
+  - Get the features for which usage data might be available.
+  - Get the metrics of a feature.
+- [Key Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Key+Usage+Api+v1.0)
+  - Get the data in a given time range for a given feature, metric and granularity with the option to filter and split data on the dimensions or DataMiner Systems.
```

Comment:
Could you clarify this sentence? It isn't clear to me what you mean with "metric and granularity" and "data on the dimensions".

Reply:
these are inputs for the api call 
![image](https://github.com/user-attachments/assets/0d38bc2c-4865-4feb-883d-619192c5ec2c)

Feature examples: Automation, Storage as a Service,...
Metric examples: Script Runs, Operations,....
Granularity options: Daily, Weekly or Monthly
Dimension splitter examples: split usage data on 'Script Name', split usage data on script 'Succeeded', ...